core/src/main/java/org/apache/calcite/rel/metadata/RelMetadataQuery.java

```diff
@@ -305,6 +305,26 @@ public static RelMetadataQuery instance() {
     }
   }
 
+  /**
+   * Returns whether the return rows of a given relational expression are empty.
```

Comment:
nit: "a given relational expression is empty"

Reply:
Let's see if there are any other opinions

Reply:
My bad, the doc is OK.